### PR TITLE
Move Measure Activity to IntroJS

### PR DESCRIPTION
**Pull request by EshaanAgg**
Fixes: [#1090](https://github.com/llaske/sugarizer/issues/1090)
I have replaced bootstrap-tour with introJs library in Measure Activity.

### CODE REVIEW

**Review by disha1202 — CHANGES_REQUESTED** [review]
This is great 👏🏻 But the tutorial is not working as it should.

[screen-recorder-sun-sep-04-2022-23-30-45.webm](https://user-images.githubusercontent.com/58461801/188327236-5e522032-f837-4ec0-9f54-4103d76b50f2.webm)

And also why have you removed bootstrap.min.css file?

File: activities/Measure.activity/index.html [hunk 1]
```diff
@@ -30,129 +29,52 @@
 			<sugar-toolitem id="activity-button"></sugar-toolitem>
 
 			<!-- Add more buttons here -->
-			<sugar-toolitem
```
[the remark this answers is not in the document]
Comment (EshaanAgg) [reply activities/Measure.activity/index.html]: The changes are probably due to using a different code formatter on my system. I have reverted them back to the original style, and also added back bootstrap.min.css which I earlier removed by mistake.

**Comment by EshaanAgg** [comment]
Also, there seems to be an error in the original bootstrap.min.css that was present in the repository. 

Original:
image
Edited:
image

Line number 7879 was earlier `background-color: #000\9`, which I have now corrected as shown.

**Comment by disha1202** [comment]
Great!! Thanks @EshaanAgg 
Looks good to me 👍🏻

**Comment by llaske** [comment]
@EshaanAgg the title bar is too small and the close button position is wrong.
It's probably related to IntroJS version see others PR on the issue to find a fix.

image

**Comment by EshaanAgg** [comment]
@llaske Downgraded the version of Introjs.css as recommended in PR #1096. Please review now.

**Comment by llaske** [comment]
That's fine now but the `test.css` file should not be there, it seems to be a dummy file.

**Comment by EshaanAgg** [comment]
Removed the same in the new commit.

**Comment by llaske** [comment]
Nice. Thanks.

**Pull request merged**